prov/psm2/src/psmx2_rma.c

```diff
@@ -973,6 +978,7 @@ ssize_t psmx2_write_generic(struct fid_ep *ep, const void *buf, size_t len,
 
 	args[0].u32w0 = 0;
 
+if (write_count++ % 810) psm2_poll(ep_priv->tx->psm2_ep);
```

Comment:
indentation and please split if statement into 2 lines.

Reply:
so is this one.